test/onigumo_test.exs

```diff
@@ -96,6 +96,17 @@ defmodule OnigumoTest do
     assert(read_output == response)
   end
 
+  test("create file name from URL") do
+    input_url = "https://onigumo.local/hello.html"
```

Comment:
The URL is inlined here instead of using `@urls`, because we need a URL that is encoded with padding. For example `bye.html` would be encoded into a string without padding and thus it would not be check that `padding: false` is used.

Reply:
OK, that's make sense when we want to ensure some behavior.
I have idea to take first url from `@urls`
```suggestion
    input_url = Enum.at(@urls, 0)
```

But your approach seems to me safer, when urls will changed in furutre and forgot to this test.